src/PluginLoader.cc

```diff
@@ -235,10 +235,10 @@ namespace gz
         return loadedPlugins;
       }
 
-      const std::string versionSymbol = "IGNCOMMONPluginAPIVersion";
-      const std::string sizeSymbol = "IGNCOMMONPluginInfoSize";
-      const std::string alignSymbol = "IGNCOMMONPluginInfoAlignment";
-      const std::string multiInfoSymbol = "IGNCOMMONMultiPluginInfo";
+      const std::string versionSymbol = "GZCOMMONPluginAPIVersion";
+      const std::string sizeSymbol = "GZCOMMONPluginInfoSize";
+      const std::string alignSymbol = "GZCOMMONPluginInfoAlignment";
+      const std::string multiInfoSymbol = "GZCOMMONMultiPluginInfo";
```

Comment:
These need to match the header:

https://github.com/gazebosim/gz-common/blob/a7149005a4ee92a6355f4c8a5192683a3cd66892/include/ignition/common/detail/PluginMacros.hh#L51-L61

We didn't migrate those because the plugin stuff is deprecated.